recipes/asn2gb/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+    name: asn2gb
+    version: "18.2"
+
+about:
+    home: https://www.ncbi.nlm.nih.gov/IEB/ToolBox/C_DOC/lxr/source/doc/asn2gb.txt
+    license: Public Domain
+    summary: asn2gb converts ASN1 format sequence records to Genbank format
+
+source:
+    fn: mac.asn2gb.gz # [osx]
+    url: ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/asn2gb/mac.asn2gb.gz # [osx]
+    sha256: e251014227d7cb9de7e33f2101b34c154d4ee40bc64b81d41dc4ed0fd5972d63 # [osx]
+    fn: linux.asn2gb.gz # [linux]
+    url: ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/asn2gb/linux.asn2gb.gz # [linux]
```

Comment:
we only compile for linux64 ... so the selector can be just [linux]
Compiling is no option?